service/awsconfig/v6/version_bundle.go

```diff
@@ -0,0 +1,55 @@
+package v6
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "aws-operator",
+				Description: "There are no changes yet, replace this with something useful.",
+				Kind:        versionbundle.KindChanged,
+			},
+		},
+		Components: []versionbundle.Component{
+			{
+				Name:    "calico",
+				Version: "3.0.2",
+			},
+			{
+				Name:    "containerlinux",
+				Version: "1576.5.0",
+			},
+			{
+				Name:    "docker",
+				Version: "17.09.0",
+			},
+			{
+				Name:    "etcd",
+				Version: "3.3.1",
+			},
+			{
+				Name:    "coredns",
+				Version: "1.0.6",
+			},
+			{
+				Name:    "kubernetes",
+				Version: "1.9.2",
+			},
+			{
+				Name:    "nginx-ingress-controller",
+				Version: "0.11.0",
+			},
+		},
+		Dependencies: []versionbundle.Dependency{},
+		Deprecated:   false,
+		Name:         "aws-operator",
+		Time:         time.Date(2018, time.February, 26, 12, 15, 0, 0, time.UTC),
+		Version:      "2.1.2",
```

Comment:
Note that I just bumped the patch level. In case this WIP bundle turns out to be a minor bump we have to sort this here.

Reply:
A patch bump makes sense to me. We can upgrade to minor if necessary.